### PR TITLE
PHPStan 1 - EasterEgg

### PR DESCRIPTION
**Pull request by lulco**

### CODE REVIEW

File: site/phpstan-latte-templates.neon [hunk 1]
```diff
@@ -4,6 +4,7 @@ parameters:
 		- app
 	level: 1
 	latte:
+		tmpDir: %currentWorkingDirectory%/temp/phpstan-latte
```
Comment (lulco) [inline site/phpstan-latte-templates.neon:7]: I'm using this for better debugging, compiled templates are directly in my application's temp, I can check them in IDE. If you don't like it, just remove this line

File: site/app/EasterEgg/NetteCve202015227Rce.php [hunk 2]
```diff
@@ -3,11 +3,14 @@
 
 namespace MichalSpacekCz\EasterEgg;
 
+/**
+ * @phpstan-type NetteCve202015227RceParameters array{command: string, eth0RxPackets: string, eth0RxBytes: string, eth0TxPackets: string, eth0TxBytes: string, eth1RxPackets: string, eth1RxBytes: string, eth1TxPackets: string, eth1TxBytes: string, loRxPackets: string, loRxBytes: string, loTxPackets: string, loTxBytes: string}
```
Comment (lulco) [inline site/app/EasterEgg/NetteCve202015227Rce.php:7]: can't find better way

**Comment by lulco** [comment]
Obviously I changed something and "classic" phpstan and tests failed, but phpstan-latte has only 11 errors O:-) :D

**Comment by lulco** [comment]
Looks like with optional parameters there is some bug in phpstan-latte https://github.com/efabrica-team/phpstan-latte/issues/370

**Comment by spaze** [comment]
Eventually, I went with "real" properties instead of array keys https://github.com/spaze/michalspacek.cz/blob/9822fc15b66a7bf6e2a8ee9ca5837c1528267d99/site/app/EasterEgg/NetteCve202015227Rce.php

**Comment by lulco** [comment]
> Eventually, I went with "real" properties instead of array keys https://github.com/spaze/michalspacek.cz/blob/9822fc15b66a7bf6e2a8ee9ca5837c1528267d99/site/app/EasterEgg/NetteCve202015227Rce.php

Yeah, that's better. I didn't want to change so much code O:-)

**Comment by spaze** [comment]
I'll close this one, thanks for the help though!

**Pull request closed without merging**